test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 //go:build !windows
+// +build !windows
```

Comment:
This is not required and not recommended. 
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives